src/mars_patcher/minimap_tile_creator.py

```diff
@@ -0,0 +1,437 @@
+from enum import Enum
+
+from mars_patcher.constants.minimap_tiles import ColoredDoor, Content, Edge, MapTile, TileEdges
+
+# Used for the edges of tiles, also used for boss icons that are always visible
+COLOR_WHITE_OUTLINE = 1
+# Used for normal connections between tiles, this appears as a wall when the
+# tile is unexplored. Also used for boss icons that disappear when explored
+COLOR_CONNECTION_BG = 2
+# Used for the background of tiles, either gray (unexplored), pink (normal),
+# or green (hidden)
+COLOR_BG = 3
+# Used for the outline of blank tiles or tiles that don't fill in the whole square
+COLOR_BLANK_OUTLINE = 4
+# Used for the background of blank tiles or tiles that don't fill in the whole square
+COLOR_BLANK_BG = 5
+# Used for the white circle/dot representing items, this hides the item when
+# the tile is unexplored
+COLOR_WHITE_ITEM = 6
+# Used for tiles with a letter in them (navigation, save, etc.)
+COLOR_YELLOW_LETTER = 7
+
+# Used for colored pixels on the edge of tiles, including hatches and the
+# outline of tiles with letters. These appear white when the tile is unexplored
+COLOR_BLUE_EDGE = 8
+COLOR_GREEN_EDGE = 9
+COLOR_RED_EDGE = 10
+COLOR_YELLOW_EDGE = 11
+
+COLOR_COUNT = 4
+
+# Used for colored pixels not on the edge of tiles, including hatches and
+# various icons. These are hidden (gray) when the tile is unexplored
+COLOR_BLUE_HIDDEN = COLOR_BLUE_EDGE + COLOR_COUNT
+COLOR_GREEN_HIDDEN = COLOR_GREEN_EDGE + COLOR_COUNT
+COLOR_RED_HIDDEN = COLOR_RED_EDGE + COLOR_COUNT
+COLOR_YELLOW_HIDDEN = COLOR_YELLOW_EDGE + COLOR_COUNT
+
+PIXELS_ITEM = [
```

Comment:
an explanation of the characters used here would be nice. wasnt immediately obvious for example that space seems to be just filler